### PR TITLE
Adding impression Viewable callback feature

### PR DESCRIPTION
**Pull request by jeffersonlicet**

### CODE REVIEW

File: js/manager.js [hunk 1]
```diff
@@ -276,6 +286,14 @@ const DFPManager = Object.assign(new EventEmitter().setMaxListeners(0), {
...
   },
 
+  attachSlotIsViewable(cb) {
+    this.on('impressionViewable', cb);
+  },
+
+  detachSlotIsViewable(cb) {
+    this.removeListener('impressionViewable', cb);
+  },
+
```
Comment (jaanauati) [inline js/manager.js]: this looks good!, just two small things:
1) can you please update the docs so to make mention of this new event and propType?
2) please revert changes made to packge.json.

thanks

Comment (jeffersonlicet) [reply js/manager.js]: I just made the changes you requested. Thank you.

**Comment by jaanauati** [comment]
@jeffersonlicet code looks good, will try to get it tested and merged by Thursday, thanks!.

**Comment by jaanauati** [comment]
this tests well on my local, merging and creating release. thanks 🍺 !

**Pull request merged**